### ISSUE TITLE
Looking for help on integrating PnP modern search extensibility samples with PnP modern search solution

### ISSUE THREAD

**Issue by GangadharMFord** [microsoft-search/pnp-modern-search-extensibility-samples#12]
Hi Team,

I'm new to development and PnP modern search solution, Kindly help us with in-detailed steps for how to integrate PnP Modern search extensibility samples with PnP modern solutions ?

I have uploaded the solution "pnp-modern-search-extensibility-demo.sppkg" at app catalog and try to add integrate custom suggestion provider with search box (Added manifest GUID to search configurations sections) but still does not working.

it more helpful for me if you provide indetailed steps on how to integrate the demo solution with PnP and then how we could add a custom web component. thank you in advance,

Regards,
gangadhar

**Comment by wobba** [comment]
Did you read https://microsoft-search.github.io/pnp-modern-search/extensibility/?

**Comment by GangadharMFord** [comment]
> Did you read https://microsoft-search.github.io/pnp-modern-search/extensibility/?

Yes, I have gone through the steps and couldn't be able to find for use GitHub extension library (Maybe I'm missed - I'm not developer).

Could you please provide us the steps to reuse the one provided in the GitHub repository.

**Comment by FranckyC** [comment]
Hi @GangadharMFord, you need to deploy the demo sppkg in your app catalog if you want to test: https://github.com/microsoft-search/pnp-modern-search-extensibility-samples/releases/tag/1.1.1. Do forget to make the solution available for all sites when deploying.

**Comment by GangadharMFord** [comment]
> Hi @GangadharMFord, you need to deploy the demo sppkg in your app catalog if you want to test: https://github.com/microsoft-search/pnp-modern-search-extensibility-samples/releases/tag/1.1.1. Do forget to make the solution available for all sites when deploying.

thank you for your message and i'm getting the below error.
![image](https://user-images.githubusercontent.com/111119555/195635488-9bcd9755-9638-4aa4-b60f-c07321cb35e4.png)

**Comment by GangadharMFord** [comment]
> > Hi @GangadharMFord, you need to deploy the demo sppkg in your app catalog if you want to test: https://github.com/microsoft-search/pnp-modern-search-extensibility-samples/releases/tag/1.1.1. Do forget to make the solution available for all sites when deploying.
> 
> thank you for your message and i'm getting the below error. ![image](https://user-images.githubusercontent.com/111119555/195635488-9bcd9755-9638-4aa4-b60f-c07321cb35e4.png)

Hi francky,

Could you please help me with this issue. I'm still facing the issue and couldn't able to integrate with search parts solution.
kindly help me with the steps. thank you.

Regards,
Gangadhar

**Comment by GangadharMFord** [comment]
> Did you read https://microsoft-search.github.io/pnp-modern-search/extensibility/?

Hi, I'm not doing any customization or added a new code into to, would like to integrate the current solution (samples demo) to the PnP search box (checking out of the box custom suggestions functionlaity) but still it was not working and has the same error.
ERROR: The custom suggestions provider with key 'CustomSuggestionsProvider' was not found. Make sure the solution is correctly deployed to the app catalog and the manifest ID registered for this Web Part.[Please refer to the official documentation.](https://microsoft-search.github.io/pnp-modern-search/)

I also packed a new samples demo solution and tested but still the issues remain same.

**Comment by wobba** [comment]
@FranckyC ?

**Comment by GangadharMFord** [comment]
@FranckyC ?
@wobba 

Could you please help me with the steps for this extensibility sample demo solution integration with search part solution?

Based on the description on the below page , I understood that the search-part 4.7 version will not support for 1.1.1. and 1.2.1 extensibility sample solution ? I can see a note with **"[Modern Search Extensibility - 1.1.1 Latest PnP Modern Search v4 - Extensibility](https://github.com/microsoft-search/pnp-modern-search-extensibility-samples/releases/tag/1.1.1)
This package is meant to be used with PnP Modern Search 4.8.0+ versions"**
Extensibility version page link: https://github.com/microsoft-search/pnp-modern-search-extensibility-samples/releases 

So currently, We using search part 4.7 version and kindly let me know which version of extensibility sample solution will support ? and SPFx version...

thank you.

**Comment by gretchunkim** [comment]
@GangadharMFord  

The extensibility works with the latest release[ 4.9.3](https://github.com/microsoft-search/pnp-modern-search/releases/tag/4.9.3 )  
![image](https://github.com/microsoft-search/pnp-modern-search-extensibility-samples/assets/8564035/95610249-f869-462e-a6c2-590ef5bea4a4)
